website/addons/sharelatex/utils.py

```diff
@@ -0,0 +1,88 @@
+import re
+import json
+import urllib
+import urllib2
+import httplib
+
+from boto import exception
```

Comment:
`boto` is a library for interacting with Amazon AWS services.  Why are you using that here?